VirtoCommerce.CatalogModule.Web/Scripts/blades/category-detail.js

```diff
@@ -34,13 +35,18 @@
         blade.title = data.name;
         blade.isLoading = false;
         blade.securityScopes = data.securityScopes;
+        blade.currentEntity.folderPath = (data.catalogId ? (getShortCatalogId(data.catalogId) + '/') : '') + data.code;
```

Comment:
This line leads to forcing category modification it is really the bug. The folder path initialization code should be moved to the CategoryImageWidget controller.